### ISSUE TITLE
undefined method `strip' for nil:NilClass

### ISSUE THREAD

**Issue by menslow** [causes/scss-lint#350]
Hi there,
I'm receiving the following error when I run scss-lint:

SCSSLint::Linter::StringQuotes raised unexpected error linting file stylesheets/patterns/_pattern-swatches.scss: 'undefined method `strip' for nil:NilClass'
/Us

Here's the Scss statement that causes the error:

```
.color {
  /**
   * Here is where we use the Sass map declared above.
   *
   * Basic syntax example:
   *
   * @each [key], [value] in [map] {
   *  .thing-#{key}
   *    color: #{value}
   * }
   *
   * For each key/value pair in the map, the @each function will run and output the code inside.
   * Neato.
   */
  @each $color-name, $color-value in $colors {
    &-#{$color-name} {
      &.swatch {
        background-color: $color-value;
      
        .color-label {
          &:before{
            content: '#{$color-value}';
          }
        }
      }
    }
  }
}
```

Specifically, it's happening with these 4 lines of the comment section:

```
   * @each [key], [value] in [map] {
   *  .thing-#{key}
   *    color: #{value}
   * }
```

If I remove these lines the linter works fine.

**Comment by sds** [comment]
Hey @menslow, thanks for the report.

Since you are using non-silent comments, Sass actually parses the contents of those comments in order to interpolate values that you may want inserted. This is part of the reason why it's [advised to use silent comments all the time](https://github.com/causes/scss-lint/blob/master/lib/scss_lint/linter/README.md#comment), as you probably didn't mean to have your comments parsed as code.

Having said that, the issue in this case is that the Sass parser also doesn't return correct source ranges for parse tree nodes in these comments, which causes the `StringQuotes` linter to fail.

I've fixed this in e9b5eea by changing `StringQuotes` to simply ignore all Sass script in comments, as I doubt any developer wants to have code in their comments linted (and also because I'm skeptical the source range problem will be fixed upstream).
